tests/test_aiohttp_security_enforcement.py

```diff
@@ -0,0 +1,126 @@
+import json
+from typing import Sequence
+
+import pytest
+from aiohttp import web
+from aiohttp.web_request import BaseRequest
+
+from apistrap.aiohttp import AioHTTPApistrap
+from apistrap.extension import OAuthFlowDefinition, OAuthSecurity
+from apistrap.schemas import ErrorResponse
+
+
+def enforcer(request: BaseRequest, scopes: Sequence[str]):
```

Comment:
What about `ensure_rw_only`, `enforcer` is a bit generic name...

Reply:
Sure!